app/assets/stylesheets/mortgage_calculator/components/_mortgagecalc.scss

```diff
@@ -93,6 +93,13 @@
   font-size: px(40);
 }
 
+@media (max-width: 425px){
```

Comment:
Can we re-use one of the media queries? `$mq-s`, `$mq-m` etc. If that solves it that would be better. If not, perhaps naming `425px` as a more descriptive variable. The reason it feels flimsy is that should the copy change, this will need to be updated again. Does this happen in Welsh too?